test/map.spec.js

```diff
@@ -4,14 +4,17 @@ import {
   isTraversable,
   getCNTORadius,
   fillGridWithSubgrid,
-  addBufferTo2dArray,
-  getSubarrayFrom2dArray,
-  getTileTraversabilityInCells,
   getMapTraversabilityInCells,
-  multiplyCorrespondingElementsAndSum,
-  convolve,
+  getTileTraversabilityInCells,
   __RewireAPI__ as MapRewireAPI,
 } from '../src/helpers/map';
+
+import { multiplyCorrespondingElementsAndSum,
+  getSubarrayFrom2dArray,
+  addBufferTo2dArray,
+  convolve } from '../src/helpers/convolve';
```

Comment:
Shouldn't there now be a `convolve.spec.js` file that tests the functions in `convolve.js`?